core/src/modules/configLoaders/index.ts

```diff
@@ -106,71 +109,75 @@ export async function processConfigObjects(
   };
   const errors: string[] = [];
 
+  configObjects = sortConfigurationObjects(configObjects);
+
   for (const i in configObjects) {
     const configObject = configObjects[i];
     if (Object.keys(configObject).length === 0) continue;
-    let klass = configObject?.class?.toLocaleLowerCase();
-    let object;
+    let klass = configObject?.class?.toLowerCase();
+    let guids: GuidsByClass;
     try {
       switch (klass) {
         case "setting":
-          object = await loadSetting(
-            configObject,
-            externallyValidate,
-            validate
-          );
+          guids = await loadSetting(configObject, externallyValidate, validate);
           break;
         case "app":
-          object = await loadApp(configObject, externallyValidate, validate);
+          guids = await loadApp(configObject, externallyValidate, validate);
           break;
         case "source":
-          object = await loadSource(configObject, externallyValidate, validate);
-          if (configObject.bootstrappedProperty) {
-            seenGuids["property"].push(
-              await validateAndFormatGuid(
-                Property,
-                configObject.bootstrappedProperty.id
-              )
-            );
-          }
+          guids = await loadSource(configObject, externallyValidate, validate);
```

Comment:
switches these to be able to return many. refactored this one to handle bootstrap too.